config/config.go

```diff
@@ -50,6 +51,7 @@ func Get() (*Config, error) {
 		HealthCheckCriticalTimeout: 90 * time.Second,
 		ZebedeeURL:                 "http://localhost:8082",
 		IsPublishing:               true,
+		DownloadServiceURL:         "http://localhost:23600",
```

Comment:
Port `23600` is the developer service - `23400` is the dp file download service according to https://github.com/ONSdigital/dp/blob/main/guides/PORTS.md

Reply:
Aha!